add blog post announcing redpoint infrared 100 list (#34)
localstack/localstack-blog.github.io: content/2024-06-18-redpoint-infrared-100/index.md

---
title: LocalStack Named to Redpoint InfraRed 100 List

> description: LocalStack, a trailblazer for cloud infrastructure emulation, has been included in the inaugural Redpoint InfraRed 100 list, highlighting the demand for cloud infrastructure development tooling. This recognition honors companies poised to lead innovation in the cloud infrastructure market and underscores LocalStack's significant impact on the industry.

Reviewer: ```suggestion
description: LocalStack, a trailblazer for cloud infrastructure emulation, has been included in the Redpoint InfraRed 100 list, highlighting the demand for cloud infrastructure development tooling. This recognition honors companies poised to lead innovation in the cloud infrastructure market and underscores LocalStack's significant impact on the industry.
```

> lead: LocalStack, a trailblazer for cloud infrastructure emulation, has been included in the inaugural Redpoint InfraRed 100 list, highlighting the demand for cloud infrastructure development tooling. This recognition honors companies poised to lead innovation in the cloud infrastructure market and underscores LocalStack's significant impact on the industry.

Reviewer: ```suggestion
lead: LocalStack, a trailblazer for cloud infrastructure emulation, has been included in the Redpoint InfraRed 100 list, highlighting the demand for cloud infrastructure development tooling. This recognition honors companies poised to lead innovation in the cloud infrastructure market and underscores LocalStack's significant impact on the industry.
```

date: 2024-06-17

> lastmod: 2024-06-17

Reviewer: Please bump the dates before publishing

100 to spotlight emerging industry leaders. The list acknowledges companies that are driving advancements and reshaping the cloud infrastructure
landscape.
<br>
<br>

{{< img-simple src="localstack-nasdaq-tower.jpg" width=300 alt="LocalStack on Nasdaq Tower">}}

<br>
<br>

LocalStack’s COO, Gerta Sheganaku, will join other InfraRed 100 leaders at Nasdaq to discuss the evolving cloud ecosystem and share insights on 
the future of the industry.

<br>

> _“We are honored to be included in the InfraRed 100 alongside such a remarkable group of companies. At LocalStack, we strive to offer developers 
> what they need and empower them with the best tools and experience possible while also supporting companies to be performant. This recognition validates
> our efforts and vision. We listen to feedback and will go above and beyond to extend this experience to more platforms. We thank Redpoint Ventures for this
> recognition and congratulate all the companies featured on the list!”_
  <div class="quote-author">
    <p><a href="https://www.linkedin.com/in/gerta-sheganaku/">Gerta Sheganaku</a>,</p>
    <p>Chief Operating Officer at LocalStack</p>
  </div>
</div>

> For the complete list of companies and to read the full InfraRed industry report [**click here**](https://www.redpoint.com/infrared/report/).

Reviewer: ```suggestion
For the complete list of companies and to read the full [InfraRed industry report](https://www.redpoint.com/infrared/report/).
```

Reviewer: `click here` is usually regarded as a bad practise since the users should know what they are clicking on.

#### About LocalStack

> [**LocalStack**](https://www.localstack.cloud/) provides a platform for local cloud development and testing, emulating a complete AWS environment on a developer’s local machine.

Reviewer: can we change this to a multi-cloud stance, which includes Snowflake & other managed providers?

This enables faster development cycles, cost savings, reduced dependency on live cloud environments, and improved testing capabilities.

> With over 52,000 GitHub stars, and 240 million Docker pulls, LocalStack has also partnered with AWS to make the tool available via its 

Reviewer: ```suggestion
With over 50K+ GitHub stars, and 250M+ Docker pulls, LocalStack has also partnered with AWS to make the tool available via its 
```

> With over 52,000 GitHub stars, and 240 million Docker pulls, LocalStack has also partnered with AWS to make the tool available via its 

Reviewer: ```suggestion
With over 52,000 GitHub stars, and 240 million Docker pulls, LocalStack has also partnered with AWS to make the tool available via the [AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-lllzw3ywntoxg?sr=0-1&ref_=beagle&applicationId=AWSMPContessa) 
```

> Marketplace for seamless integration with existing business needs.

Reviewer: ```suggestion
for seamless integration with existing business needs.
```

#### Resources

- [**Sign up**](https://app.localstack.cloud/sign-up) for a free account and get started with LocalStack
- [**LocalStack Documentation**](https://docs.localstack.cloud/)
- [**LocalStack Developer Hub**](https://docs.localstack.cloud/developer-hub/)
- [**Contact LocalStack and Community**](https://www.localstack.cloud/contact)

> 

Reviewer: Add a `Follow us on LinkedIn, Twitter/X, ... etc`
